File: supabase/functions/analyze-food/classify.ts
// Get food category for similar item matching
export function getFoodCategory(foodName: string, ingredients: string = ''): string {
  const name = foodName.toLowerCase();
  const ing = ingredients.toLowerCase();
  
  // Sweet/Dessert items
  if (name.includes('chocolate') || name.includes('candy') || name.includes('cookie') || 
      name.includes('cake') || name.includes('ice cream') || name.includes('dessert') ||
      name.includes('sweet') || name.includes('brownie') || name.includes('pastry') ||
      name.includes('muffin') || name.includes('donut') || (ing.includes('sugar') && ing.includes('cocoa'))) {
    return 'sweet';
  }
  
  // Salty snacks
  if (name.includes('chip') || name.includes('crisp') || name.includes('cracker') ||
      name.includes('pretzel') || name.includes('popcorn') || name.includes('snack') ||
      name.includes('namkeen') || name.includes('mixture')) {
    return 'salty-snack';
  }
  
  // Beverages
  if (name.includes('soda') || name.includes('juice') || name.includes('drink') ||
      name.includes('cola') || name.includes('beverage') || name.includes('tea') || 
      name.includes('coffee') || name.includes('milkshake') || name.includes('smoothie')) {
    return 'beverage';
  }
  
  // Fried foods
  if (name.includes('fried') || name.includes('fries') || name.includes('nugget') ||
      name.includes('tempura') || name.includes('pakora') || name.includes('samosa') ||
      name.includes('bhaji') || name.includes('vada')) {
    return 'fried';
  }
  
  // Fast food
  if (name.includes('burger') || name.includes('pizza') || name.includes('sandwich') ||
      name.includes('hot dog') || name.includes('wrap') || name.includes('taco')) {
    return 'fast-food';
  }
  
  // Main meals - rice/grain based
  if (name.includes('rice') || name.includes('biryani') || name.includes('pulao') ||
      name.includes('fried rice') || name.includes('khichdi')) {
    return 'rice-dish';
  }
  
  // Curry/gravy dishes
  if (name.includes('curry') || name.includes('masala') || name.includes('korma') ||
      name.includes('tikka') || name.includes('butter chicken') || name.includes('paneer')) {
    return 'curry';
  }
  
  // Pasta/Noodles
  if (name.includes('pasta') || name.includes('noodle') || name.includes('spaghetti') ||
      name.includes('lasagna') || name.includes('chow mein') || name.includes('hakka')) {
    return 'pasta-noodles';
  }
  
  // Breakfast items
  if (name.includes('cereal') || name.includes('oatmeal') || name.includes('pancake') ||
      name.includes('waffle') || name.includes('idli') || name.includes('dosa') ||
      name.includes('poha') || name.includes('upma') || name.includes('paratha')) {
    return 'breakfast';
  }
  
  return 'general';
}

// Detect meal type based on food name
export function detectMealType(foodName: string): 'breakfast' | 'lunch-dinner' | 'snack' | 'any' {
  const name = foodName.toLowerCase();
  
  // Breakfast items
  if (name.includes('cereal') || name.includes('oatmeal') || name.includes('porridge') ||
      name.includes('pancake') || name.includes('waffle') || name.includes('toast') ||
      name.includes('egg') || name.includes('bacon') || name.includes('sausage') ||
      name.includes('paratha') || name.includes('idli') || name.includes('dosa') ||
      name.includes('poha') || name.includes('upma') || name.includes('cornflakes') ||
      name.includes('muesli') || name.includes('granola') || name.includes('breakfast')) {
    return 'breakfast';
  }
  
  // Lunch/Dinner items (main meals)
  if (name.includes('rice') || name.includes('biryani') || name.includes('curry') ||
      name.includes('dal') || name.includes('roti') || name.includes('naan') ||
      name.includes('pasta') || name.includes('noodle') || name.includes('stir fry') ||
      name.includes('steak') || name.includes('chicken') || name.includes('fish') ||
      name.includes('salmon') || name.includes('burger') || name.includes('pizza') ||
      name.includes('sandwich') || name.includes('wrap') || name.includes('salad') ||
      name.includes('soup') || name.includes('thali') || name.includes('paneer') ||
      name.includes('tikka') || name.includes('kebab') || name.includes('pulao') ||
      name.includes('fried rice') || name.includes('chow mein') || name.includes('manchurian') ||
      name.includes('korma') || name.includes('masala') || name.includes('vindaloo') ||
      name.includes('lasagna') || name.includes('casserole') || name.includes('roast')) {
    return 'lunch-dinner';
  }
  
  // Snack items
  if (name.includes('chip') || name.includes('crisp') || name.includes('cookie') ||
      name.includes('biscuit') || name.includes('chocolate') || name.includes('candy') ||
      name.includes('popcorn') || name.includes('nuts') || name.includes('cracker') ||
      name.includes('bar') || name.includes('samosa') || name.includes('pakora') ||
      name.includes('bhaji') || name.includes('vada') || name.includes('chaat') ||
      name.includes('ice cream') || name.includes('cake') || name.includes('pastry') ||
      name.includes('muffin') || name.includes('donut') || name.includes('brownie')) {
    return 'snack';
  }
  
  return 'any';
}

// Get search terms that match the same food category - IMPROVED for better suggestions
export function getCategoryMatchedSearchTerms(foodName: string, category: string, mealType: string): string[] {
  const lowerName = foodName.toLowerCase();
  
  // INDIAN CURRY/GRAVY dishes - suggest healthier Indian curries
  if (lowerName.includes('butter chicken') || lowerName.includes('chicken curry') ||
      lowerName.includes('tikka masala') || lowerName.includes('korma')) {
    return ['tandoori chicken', 'chicken tikka grilled', 'chicken kebab', 'grilled chicken breast'];
  }
  
  if (lowerName.includes('paneer') || lowerName.includes('palak') || lowerName.includes('matar')) {
    return ['tofu curry', 'grilled paneer tikka', 'palak tofu', 'cottage cheese grilled'];
  }
  
  if (lowerName.includes('dal') || lowerName.includes('lentil') || lowerName.includes('rajma') ||
      lowerName.includes('chole') || lowerName.includes('chana')) {
    return ['moong dal soup', 'masoor dal', 'sprouted lentils', 'chickpea salad'];
  }
  
  // RICE dishes - suggest healthier rice alternatives
  if (lowerName.includes('biryani') || lowerName.includes('pulao') || lowerName.includes('fried rice')) {
    return ['brown rice pulao', 'quinoa biryani', 'vegetable khichdi', 'millets pulao'];
  }
  
  if (lowerName.includes('rice') && !lowerName.includes('fried')) {
    return ['brown rice', 'quinoa', 'millets cooked', 'cauliflower rice'];
  }
  
  // BREAD/ROTI - suggest healthier flatbreads
  if (lowerName.includes('naan') || lowerName.includes('paratha') || lowerName.includes('kulcha') ||
      lowerName.includes('bhatura') || lowerName.includes('poori') || lowerName.includes('puri')) {
    return ['whole wheat roti', 'multigrain roti', 'bajra roti', 'jowar roti'];
  }
  
  if (lowerName.includes('roti') || lowerName.includes('chapati') || lowerName.includes('bread')) {
    return ['multigrain bread', 'oats roti', 'ragi roti', 'whole wheat sourdough'];
  }
  
  // FRIED SNACKS - suggest baked/grilled versions
  if (lowerName.includes('samosa') || lowerName.includes('pakora') || lowerName.includes('pakoda') ||
      lowerName.includes('bhaji') || lowerName.includes('vada') || lowerName.includes('kachori')) {
    return ['baked samosa', 'air fried pakora', 'grilled paneer tikka', 'roasted chickpeas'];
  }
  
  if (lowerName.includes('fries') || lowerName.includes('french fries')) {
    return ['baked sweet potato fries', 'air fried potato wedges', 'zucchini fries baked', 'carrot fries'];
  }
  
  // CHICKEN/MEAT - suggest grilled/baked versions
  if (lowerName.includes('fried chicken') || lowerName.includes('chicken nuggets') ||
      lowerName.includes('chicken wings')) {
    return ['grilled chicken breast', 'baked chicken', 'tandoori chicken', 'chicken kebab'];
  }
  
  if (lowerName.includes('chicken') || lowerName.includes('meat') || lowerName.includes('mutton') ||
      lowerName.includes('lamb') || lowerName.includes('beef')) {
    return ['grilled chicken breast', 'lean turkey', 'grilled fish fillet', 'baked salmon'];
  }
  
  // FISH/SEAFOOD - suggest healthier preparations
  if (lowerName.includes('fish fry') || lowerName.includes('fried fish')) {
    return ['grilled fish', 'baked salmon', 'steamed fish', 'fish tikka'];
  }
  
  if (lowerName.includes('fish') || lowerName.includes('salmon') || lowerName.includes('prawn') ||
      lowerName.includes('shrimp')) {
    return ['grilled salmon', 'steamed fish', 'baked cod', 'grilled prawns'];
  }
  
  // PIZZA/BURGER - suggest healthier versions
  if (lowerName.includes('pizza')) {
    return ['whole wheat pizza', 'cauliflower crust pizza', 'grilled vegetable wrap', 'stuffed bell peppers'];
  }
  
  if (lowerName.includes('burger')) {
    return ['grilled chicken sandwich', 'lettuce wrap burger', 'turkey burger', 'veggie burger'];
  }
  
  // PASTA/NOODLES - suggest whole grain versions
  if (lowerName.includes('pasta') || lowerName.includes('macaroni') || lowerName.includes('spaghetti') ||
      lowerName.includes('lasagna')) {
    return ['whole wheat pasta', 'chickpea pasta', 'lentil pasta', 'zucchini noodles'];
  }
  
  if (lowerName.includes('noodle') || lowerName.includes('chow mein') || lowerName.includes('hakka')) {
    return ['soba noodles', 'rice noodles', 'vegetable stir fry', 'zucchini noodles'];
  }
  
  // SWEETS/DESSERTS - suggest healthier alternatives
  if (lowerName.includes('gulab jamun') || lowerName.includes('rasgulla') || lowerName.includes('jalebi') ||
      lowerName.includes('ladoo') || lowerName.includes('barfi') || lowerName.includes('halwa')) {
    return ['dates ladoo', 'ragi ladoo', 'oats ladoo', 'dry fruit barfi'];
  }
  
  if (lowerName.includes('cake') || lowerName.includes('pastry') || lowerName.includes('brownie') ||
      lowerName.includes('muffin') || lowerName.includes('donut')) {
    return ['banana bread', 'oats muffin', 'protein brownie', 'date walnut cake'];
  }
  
  if (lowerName.includes('ice cream') || lowerName.includes('kulfi')) {
    return ['frozen yogurt', 'banana nice cream', 'coconut ice cream', 'fruit sorbet'];
  }
  
  if (lowerName.includes('chocolate') || lowerName.includes('candy')) {
    return ['dark chocolate 85%', 'cacao nibs', 'dates chocolate', 'almond butter cups'];
  }
  
  // COOKIES/BISCUITS - suggest healthier versions
  if (lowerName.includes('cookie') || lowerName.includes('biscuit') || lowerName.includes('cream')) {
    return ['oats cookies', 'ragi biscuits', 'multigrain digestive', 'almond cookies'];
  }
  
  // CHIPS/SNACKS - suggest baked alternatives with variety
  if (lowerName.includes('chip') || lowerName.includes('crisp') || lowerName.includes('namkeen') ||
      lowerName.includes('lay') || lowerName.includes('kurkure') || lowerName.includes('bhujia')) {
    return ['roasted makhana fox nuts', 'roasted chana masala', 'baked multigrain chips', 'masala peanuts roasted'];
  }
  
  // PACKAGED SNACKS specifically
  if (lowerName.includes('maggi') || lowerName.includes('instant noodle') || lowerName.includes('cup noodle')) {
    return ['oats noodles', 'vegetable hakka noodles', 'rice noodles', 'soba noodles'];
  }
  
  // BEVERAGES - suggest healthier drinks
  if (lowerName.includes('soda') || lowerName.includes('cola') || lowerName.includes('soft drink') ||
      lowerName.includes('pepsi') || lowerName.includes('coke')) {
    return ['coconut water', 'lime water', 'green tea', 'buttermilk'];
  }
  
  if (lowerName.includes('milkshake') || lowerName.includes('shake') || lowerName.includes('smoothie')) {
    return ['protein smoothie', 'banana oat smoothie', 'green smoothie', 'yogurt smoothie'];
  }
  
  if (lowerName.includes('juice') || lowerName.includes('drink')) {
    return ['coconut water', 'vegetable juice', 'infused water', 'green tea'];
  }
  
  // BREAKFAST items
  if (lowerName.includes('dosa') || lowerName.includes('idli') || lowerName.includes('uttapam')) {
    return ['ragi dosa', 'oats idli', 'moong dal chilla', 'vegetable uttapam'];
  }
  
  if (lowerName.includes('poha') || lowerName.includes('upma') || lowerName.includes('paratha')) {
    return ['vegetable poha', 'oats upma', 'multigrain paratha', 'sprouts paratha'];
  }
  
  if (lowerName.includes('cereal') || lowerName.includes('cornflakes')) {
    return ['oatmeal', 'muesli', 'quinoa porridge', 'chia pudding'];
  }
  
  // EGG dishes
  if (lowerName.includes('egg') || lowerName.includes('omelette') || lowerName.includes('omelet')) {
    return ['egg white omelette', 'boiled eggs', 'poached eggs', 'scrambled egg whites'];
  }
  
  // FRUIT - suggest similar fruits
  if (lowerName.includes('banana') || lowerName.includes('apple') || lowerName.includes('mango') ||
      lowerName.includes('orange') || lowerName.includes('fruit')) {
    return ['berries mixed', 'papaya', 'guava', 'watermelon'];
  }
  
  // Meal-type based fallback
  if (mealType === 'breakfast') {
    return ['oats porridge', 'moong dal chilla', 'vegetable poha', 'egg white omelette'];
  } else if (mealType === 'lunch-dinner') {
    return ['grilled chicken breast', 'dal tadka', 'vegetable stir fry', 'quinoa bowl'];
  }
  
  // Default healthy snacks
  return ['mixed nuts', 'roasted chickpeas', 'greek yogurt', 'fresh fruit bowl'];
}

File: python_app/classify.py
from typing import List

def get_food_category(food_name: str, ingredients: str = '') -> str:
    name = food_name.lower()
    ing = ingredients.lower()
    
    # Sweet/Dessert items
    if any(k in name for k in [
        'chocolate', 'candy', 'cookie', 'cake', 'ice cream', 'dessert', 'sweet', 'brownie', 
        'pastry', 'muffin', 'donut', 'gulab jamun', 'rasgulla', 'jalebi', 'ladoo', 'barfi', 
        'halwa', 'kheer', 'kulfi', 'rasmalai', 'peda', 'mysore pak'
    ]) or ('sugar' in ing and 'cocoa' in ing):
        return 'sweet'
        
    # Salty snacks
    if any(k in name for k in ['chip', 'crisp', 'cracker', 'pretzel', 'popcorn', 'snack', 'namkeen', 'mixture', 'bhujia', 'sev', 'makhana']):
        return 'salty-snack'
        
    # Beverages
    if any(k in name for k in ['soda', 'juice', 'drink', 'cola', 'beverage', 'tea', 'coffee', 'milkshake', 'smoothie', 'lassi', 'chaas']):
        return 'beverage'
        
    # Fried foods
    if any(k in name for k in ['fried', 'fries', 'nugget', 'tempura', 'pakora', 'pakoda', 'samosa', 'bhaji', 'vada', 'kachori', 'poori', 'bhatura']):
        return 'fried'
        
    # Fast food
    if any(k in name for k in ['burger', 'pizza', 'sandwich', 'hot dog', 'wrap', 'taco', 'roll', 'frankie']):
        return 'fast-food'
        
    # Main meals - rice/grain based
    if any(k in name for k in ['rice', 'biryani', 'pulao', 'fried rice', 'khichdi', 'curd rice']):
        return 'rice-dish'
        
    # Curry/gravy dishes
    if any(k in name for k in ['curry', 'masala', 'korma', 'tikka', 'butter chicken', 'paneer', 'dal', 'sambar', 'rajma', 'chole']):
        return 'curry'
        
    # Pasta/Noodles
    if any(k in name for k in ['pasta', 'noodle', 'spaghetti', 'lasagna', 'chow mein', 'hakka', 'maggi']):
        return 'pasta-noodles'
        
    # Breakfast items
    if any(k in name for k in ['cereal', 'oatmeal', 'pancake', 'waffle', 'idli', 'dosa', 'poha', 'upma', 'paratha', 'chilla', 'uttapam']):
        return 'breakfast'
        
    return 'general'

def detect_meal_type(food_name: str) -> str:
    name = food_name.lower()
    
    # Breakfast items
    if any(k in name for k in [
        'cereal', 'oatmeal', 'porridge', 'pancake', 'waffle', 'toast', 'egg', 'bacon', 
        'sausage', 'paratha', 'idli', 'dosa', 'poha', 'upma', 'cornflakes', 'muesli', 'granola', 'breakfast', 'chilla', 'uttapam'
    ]):
        return 'breakfast'
        
    # Lunch/Dinner items (main meals)
    if any(k in name for k in [
        'rice', 'biryani', 'curry', 'dal', 'roti', 'naan', 'pasta', 'noodle', 'stir fry',
        'steak', 'chicken', 'fish', 'salmon', 'burger', 'pizza', 'sandwich', 'wrap', 'salad',
        'soup', 'thali', 'paneer', 'tikka', 'kebab', 'pulao', 'fried rice', 'chow mein', 'manchurian',
        'korma', 'masala', 'vindaloo', 'lasagna', 'casserole', 'roast'
    ]):
        return 'lunch-dinner'
        
    # Snack items
    if any(k in name for k in [
        'chip', 'crisp', 'cookie', 'biscuit', 'chocolate', 'candy', 'popcorn', 'nuts', 'cracker',
        'bar', 'samosa', 'pakora', 'bhaji', 'vada', 'chaat', 'ice cream', 'cake', 'pastry',
        'muffin', 'donut', 'brownie', 'namkeen', 'makhana', 'seeds', 'snack'
    ]):
        return 'snack'
        
    return 'any'

def get_category_matched_search_terms(food_name: str, category: str, meal_type: str) -> List[str]:
    lower_name = food_name.lower()
    
    # Indian curry / gravy dishes
    if any(k in lower_name for k in ['butter chicken', 'chicken curry', 'tikka masala', 'korma']):
        return ['tandoori chicken', 'chicken tikka grilled', 'chicken kebab', 'grilled chicken breast']
        
    if any(k in lower_name for k in ['paneer', 'palak', 'matar']):
        return ['tofu curry', 'grilled paneer tikka', 'palak tofu', 'cottage cheese grilled']
        
    if any(k in lower_name for k in ['dal', 'lentil', 'rajma', 'chole', 'chana']):
        return ['moong dal soup', 'masoor dal', 'sprouted lentils', 'chickpea salad']
        
    # Rice dishes
    if any(k in lower_name for k in ['biryani', 'pulao', 'fried rice']):
        return ['brown rice pulao', 'quinoa biryani', 'vegetable khichdi', 'millets pulao']
        
    if 'rice' in lower_name and 'fried' not in lower_name:
        return ['brown rice', 'quinoa', 'millets cooked', 'cauliflower rice']
        
    # Bread / Roti
    if any(k in lower_name for k in ['naan', 'paratha', 'kulcha', 'bhatura', 'poori', 'puri']):
        return ['whole wheat roti', 'multigrain roti', 'bajra roti', 'jowar roti']
        
    if any(k in lower_name for k in ['roti', 'chapati', 'bread']):
        return ['multigrain bread', 'oats roti', 'ragi roti', 'whole wheat sourdough']
        
    # Fried snacks
    if any(k in lower_name for k in ['samosa', 'pakora', 'pakoda', 'bhaji', 'vada', 'kachori']):
        return ['baked samosa', 'air fried pakora', 'grilled paneer tikka', 'roasted chickpeas']
        
    if any(k in lower_name for k in ['fries', 'french fries']):
        return ['baked sweet potato fries', 'air fried potato wedges', 'zucchini fries baked', 'carrot fries']
        
    # Chicken / Meat
    if any(k in lower_name for k in ['fried chicken', 'chicken nuggets', 'chicken wings']):
        return ['grilled chicken breast', 'baked chicken', 'tandoori chicken', 'chicken kebab']
        
    if any(k in lower_name for k in ['chicken', 'meat', 'mutton', 'lamb', 'beef']):
        return ['grilled chicken breast', 'lean turkey', 'grilled fish fillet', 'baked salmon']
        
    # Fish / Seafood
    if any(k in lower_name for k in ['fish fry', 'fried fish']):
        return ['grilled fish', 'baked salmon', 'steamed fish', 'fish tikka']
        
    if any(k in lower_name for k in ['fish', 'salmon', 'prawn', 'shrimp']):
        return ['grilled salmon', 'steamed fish', 'baked cod', 'grilled prawns']
        
    # Pizza / Burger
    if 'pizza' in lower_name:
        return ['whole wheat pizza', 'cauliflower crust pizza', 'grilled vegetable wrap', 'stuffed bell peppers']
        
    if 'burger' in lower_name:
        return ['grilled chicken sandwich', 'lettuce wrap burger', 'turkey burger', 'veggie burger']
        
    # Pasta / Noodles
    if any(k in lower_name for k in ['pasta', 'macaroni', 'spaghetti', 'lasagna']):
        return ['whole wheat pasta', 'chickpea pasta', 'lentil pasta', 'zucchini noodles']
        
    if any(k in lower_name for k in ['noodle', 'chow mein', 'hakka', 'maggi']):
        return ['soba noodles', 'rice noodles', 'vegetable stir fry', 'zucchini noodles']
        
    # Sweets / Desserts
    if any(k in lower_name for k in ['gulab jamun', 'rasgulla', 'jalebi', 'ladoo', 'barfi', 'halwa']):
        return ['dates ladoo', 'ragi ladoo', 'oats ladoo', 'dry fruit barfi']
        
    if any(k in lower_name for k in ['cake', 'pastry', 'brownie', 'muffin', 'donut']):
        return ['banana bread', 'oats muffin', 'protein brownie', 'date walnut cake']
        
    if any(k in lower_name for k in ['ice cream', 'kulfi']):
        return ['frozen yogurt', 'banana nice cream', 'coconut ice cream', 'fruit sorbet']
        
    if any(k in lower_name for k in ['chocolate', 'candy']):
        return ['dark chocolate 85%', 'cacao nibs', 'dates chocolate', 'almond butter cups']
        
    # Cookies / Biscuits
    if any(k in lower_name for k in ['cookie', 'biscuit', 'cream']):
        return ['oats cookies', 'ragi biscuits', 'multigrain digestive', 'almond cookies']
        
    # Chips / Snacks
    if any(k in lower_name for k in ['chip', 'crisp', 'namkeen', 'lay', 'kurkure', 'bhujia']):
        return ['roasted makhana fox nuts', 'roasted chana masala', 'baked multigrain chips', 'masala peanuts roasted']
        
    # Beverages
    if any(k in lower_name for k in ['soda', 'cola', 'soft drink', 'pepsi', 'coke']):
        return ['coconut water', 'lime water', 'green tea', 'buttermilk']
        
    if any(k in lower_name for k in ['milkshake', 'shake', 'smoothie']):
        return ['protein smoothie', 'banana oat smoothie', 'green smoothie', 'yogurt smoothie']
        
    if any(k in lower_name for k in ['juice', 'drink']):
        return ['coconut water', 'vegetable juice', 'infused water', 'green tea']
        
    # Breakfast
    if any(k in lower_name for k in ['dosa', 'idli', 'uttapam']):
        return ['ragi dosa', 'oats idli', 'moong dal chilla', 'vegetable uttapam']
        
    if any(k in lower_name for k in ['poha', 'upma', 'paratha']):
        return ['vegetable poha', 'oats upma', 'multigrain paratha', 'sprouts paratha']
        
    if any(k in lower_name for k in ['cereal', 'cornflakes']):
        return ['oatmeal', 'muesli', 'quinoa porridge', 'chia pudding']
        
    # Eggs
    if any(k in lower_name for k in ['egg', 'omelette', 'omelet']):
        return ['egg white omelette', 'boiled eggs', 'poached eggs', 'scrambled egg whites']
        
    # Fruit
    if any(k in lower_name for k in ['banana', 'apple', 'mango', 'orange', 'fruit']):
        return ['berries mixed', 'papaya', 'guava', 'watermelon']
        
    # Meal-type based fallback
    if meal_type == 'breakfast':
        return ['oats porridge', 'moong dal chilla', 'vegetable poha', 'egg white omelette']
    elif meal_type == 'lunch-dinner':
        return ['grilled chicken breast', 'dal tadka', 'vegetable stir fry', 'quinoa bowl']
        
    # Default healthy snacks
    return ['mixed nuts', 'roasted chickpeas', 'greek yogurt', 'fresh fruit bowl']
